sdk/loadtestservice/azure-mgmt-loadtestservice/tests/test_loadtestresource_operations.py

```diff
@@ -0,0 +1,130 @@
+# coding: utf-8
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+
+import unittest
+
+from azure.mgmt.loadtestservice import LoadTestMgmtClient
+from azure.mgmt.loadtestservice.models import LoadTestResource, LoadTestResourcePatchRequestBody, QuotaBucketRequest
+from devtools_testutils import AzureMgmtRecordedTestCase, recorded_by_proxy, ResourceGroupPreparer
+
+AZURE_LOCATION = 'westus2'
+LOAD_TEST_RESOURCE_NAME = 'py-sdk-testing-resource-cp'
+
+class TestLoadTestResourceOperations(AzureMgmtRecordedTestCase):
+    
+    def setup_method(self, method):
+        self.loadtestservice_client = self.create_mgmt_client(
+            LoadTestMgmtClient
+        )
+    
+    @ResourceGroupPreparer()
+    @recorded_by_proxy
+    def test_load_test_resource_operations(self, resource_group):
+
+        # Create a load test resource
+        loadtestresource_create_payload = LoadTestResource(location=AZURE_LOCATION)
+        
+        # Create a load test resource create begin - returns a poller
+        loadtest_resource_poller = self.loadtestservice_client.load_tests.begin_create_or_update(resource_group.name, LOAD_TEST_RESOURCE_NAME, loadtestresource_create_payload)
+        
+        # Wait for the poller to complete
+        while loadtest_resource_poller.done()==False:
+            loadtest_resource_poller.wait(2)
```

Comment:
You can try `loadtest_resource_poller = xxx.xxx.begin_create_or_update(...).result()` which is cleaner

Reply:
@msyyc Quoting from the description of .result() method:
`"""Return the result of the long running operation, or
        the result available after the specified timeout.`

We would need to continue only after LRO has completed, and .result() might produce results even if LRO has not completed. I checked if there are any other inbuilt methods to await LRO completion but couldn't find any.
that is why I had implemented this while condition.

Please let me know if there is any other better way to achieve the same.